src/ILLink.RoslynAnalyzer/DynamicallyAccessedMembersAnalyzer.cs

```diff
@@ -59,6 +61,30 @@ public override void Initialize (AnalysisContext context)
 				var invocationOperation = (IInvocationOperation) context.Operation;
 				ProcessInvocationOperation (context, invocationOperation);
 			}, OperationKind.Invocation);
+			context.RegisterSyntaxNodeAction (context => {
+				ProcessSyntaxNode (context);
+			}, SyntaxKind.GenericName);
+		}
+
+		private void ProcessSyntaxNode (SyntaxNodeAnalysisContext context)
```

Comment:
This is a bit vague, what exactly does this method process?

Reply:
Fixed